### PR TITLE
added 2 C programs on LeetCode Problem no. 100 & 102

### PR DESCRIPTION
**Pull request by arnabpal2022**
#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

File: leetcode/src/100.c [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
```
Comment (Panquesito7) [inline leetcode/src/100.c:1]:
```suggestion
```

**Comment by Panquesito7** [comment]
Seems the CI is failing because you added two files in the same directory.
Please make another PR for one of the LeetCode problems. If you need any help, let us know. 🙂

**Comment by Panquesito7** [comment]
I'm not 100% sure what's the cause for the CI failing. We can try removing one file and adding it in a separate PR. 🙂

**Comment by arnabpal2022** [comment]
@Panquesito7 Can I Close This One and Make Another??

**Comment by Panquesito7** [comment]
> @Panquesito7 Can I Close This One and Make Another??

The best thing to do would be to remove one file or another.
Let's try removing the `102.c` file and make a separate PR for it _only_. 🙂

**Pull request closed without merging**